README.md

```diff
@@ -245,43 +108,23 @@ Feel free to check the [issues page](../../issues/).
 
```

Comment:
[OPTIONAL] 👍🏽 

- I think it will be great if you added a link to the issues page of your repo, just incase someone wants to leave some suggestions. 🤗